### PR TITLE
Catalog. Add task solution

### PR DESCRIPTION
**Pull request by khviktoriia**
- [DEMO LINK](https://khviktoriia.github.io/layout_catalog/)
- [TEST REPORT LINK](https://khviktoriia.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/nav.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    height: 100%;
+    align-items: center;
+    font-weight: 500;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/nav.scss:17]: This is the default value. No need to write it
```suggestion
```

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+@import "blocks/card";
+@import "blocks/stars";
+
+/*body {
+  position: relative;
+  top: 0;
+  min-width: 100px;
+  min-height: 100px;
+  border-bottom: 1px;
+}*/
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:15]: Remove these comments

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
But let's make your code perfect

File: src/styles/blocks/nav.scss (hunk 1 above)
Comment (khviktoriia) [reply src/styles/blocks/nav.scss:17]: The default value is 400, isn`t it?

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__link {
+    display: flex;
+    box-sizing: border-box;
```
Comment (OleziO) [inline src/styles/blocks/card.scss:67]: Move to * selector and remove from all other places.

**Review by OleziO — CHANGES_REQUESTED** [review]

**Review by volodymyr-soltys97 — APPROVED** [review]
Nice work 🔥

**Pull request closed without merging**